sensu.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency "em-redis-unified", "1.0.0"
   s.add_dependency "sinatra", "1.4.6"
   s.add_dependency "async_sinatra", "1.2.0"
-  s.add_dependency "thin", "1.6.3" unless RUBY_PLATFORM =~ /java/
```

Comment:
Ah, I apologize that I missed you're building a java-specific gem

- https://rubygems.org/gems/sensu/versions/0.20.5-java
- https://rubygems.org/gems/sensu/versions/0.20.5

I expected to see something in the source code (`ext/sensu/extconf.rb`) and didn't think to look at RubyGems.

Not sure what your build process is. Are you using https://github.com/rake-compiler/rake-compiler-dock or https://github.com/tjschuck/rake-compiler-dev-box ( https://github.com/rake-compiler/rake-compiler )? I love that workflow.

Then you could just modify the thin dep in your buildstep, e.g. https://github.com/codahale/bcrypt-ruby/blob/1f9184a8df2b90fa02d01a32a364eefc9072e1b9/Rakefile#L49-L59